### DISCUSSION TITLE
what cause this error？

### DISCUSSION THREAD

**Discussion by BelugaStd** [magicpdf/Magic-PDF#309] [Q&A]
分析错误(MinerU) E:\UsefulTool\MinerU-magic_pdf-0.6.2b1-released>magic-pdf pdf-command --pdf "D:\\1.pdf" --inside_model true
2024-08-03 09:57:55.458 | ERROR    | magic_pdf.cli.magicpdf:parse_doc:338 - Invalid control character at: line 6 column 29 (char 156)
Traceback (most recent call last):

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
           │         │     └ {'__name__': '__main__', '__doc__': None, '__package__': '', '__loader__': <zipimporter object "C:\Users\Beluga\.conda\envs\M...
           │         └ <code object <module> at 0x000001C184D37EC0, file "C:\Users\Beluga\.conda\envs\MinerU\Scripts\magic-pdf.exe\__main__.py", lin...
           └ <function _run_code at 0x000001C184D21480>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
         │     └ {'__name__': '__main__', '__doc__': None, '__package__': '', '__loader__': <zipimporter object "C:\Users\Beluga\.conda\envs\M...
         └ <code object <module> at 0x000001C184D37EC0, file "C:\Users\Beluga\.conda\envs\MinerU\Scripts\magic-pdf.exe\__main__.py", lin...

  File "C:\Users\Beluga\.conda\envs\MinerU\Scripts\magic-pdf.exe\__main__.py", line 7, in <module>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\site-packages\click\core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x000001C185186680>
           └ <Group cli>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\site-packages\click\core.py", line 1078, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x000001C184D88FA0>
         │    └ <function MultiCommand.invoke at 0x000001C185187640>
         └ <Group cli>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\site-packages\click\core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x000001C1DBAAB2E0>
           │               │       │       └ <function Command.invoke at 0x000001C185187130>
           │               │       └ <Command pdf-command>
           │               └ <click.core.Context object at 0x000001C1DBAAB2E0>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x000001C184AA7D00>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\site-packages\click\core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'pdf': 'D:\\\\1.pdf', 'inside_model': True, 'model': None, 'method': 'auto', 'model_mode': 'full'}
           │   │      │    │           └ <click.core.Context object at 0x000001C1DBAAB2E0>
           │   │      │    └ <function pdf_command at 0x000001C1DBAD4B80>
           │   │      └ <Command pdf-command>
           │   └ <function Context.invoke at 0x000001C185185EA0>
           └ <click.core.Context object at 0x000001C1DBAAB2E0>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\site-packages\click\core.py", line 783, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'pdf': 'D:\\\\1.pdf', 'inside_model': True, 'model': None, 'method': 'auto', 'model_mode': 'full'}
                       └ ()

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\site-packages\magic_pdf\cli\magicpdf.py", line 352, in pdf_command
    parse_doc(pdf)
    │         └ 'D:\\\\1.pdf'
    └ <function pdf_command.<locals>.parse_doc at 0x000001C1DBAD48B0>

> File "C:\Users\Beluga\.conda\envs\MinerU\lib\site-packages\magic_pdf\cli\magicpdf.py", line 328, in parse_doc
    jso = json_parse.loads(get_model_json(model, doc_path))
          │          │     │              │      └ 'D:\\\\1.pdf'
          │          │     │              └ None
          │          │     └ <function pdf_command.<locals>.get_model_json at 0x000001C1DBAD4670>
          │          └ <function loads at 0x000001C184D9C820>
          └ <module 'json' from 'C:\\Users\\Beluga\\.conda\\envs\\MinerU\\lib\\json\\__init__.py'>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           │                │      └ '{\n    "bucket_info":{\n        "bucket-name-1":["ak", "sk", "endpoint"],\n        "bucket-name-2":["ak", "sk", "endpoint"]\...
           │                └ <function JSONDecoder.decode at 0x000001C184D9C0D0>
           └ <json.decoder.JSONDecoder object at 0x000001C184D89000>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               │    │          │      │  └ '{\n    "bucket_info":{\n        "bucket-name-1":["ak", "sk", "endpoint"],\n        "bucket-name-2":["ak", "sk", "endpoint"]\...
               │    │          │      └ <built-in method match of re.Pattern object at 0x000001C184963510>
               │    │          └ '{\n    "bucket_info":{\n        "bucket-name-1":["ak", "sk", "endpoint"],\n        "bucket-name-2":["ak", "sk", "endpoint"]\...
               │    └ <function JSONDecoder.raw_decode at 0x000001C184D9C160>
               └ <json.decoder.JSONDecoder object at 0x000001C184D89000>

  File "C:\Users\Beluga\.conda\envs\MinerU\lib\json\decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
               │    │         │  └ 0
               │    │         └ '{\n    "bucket_info":{\n        "bucket-name-1":["ak", "sk", "endpoint"],\n        "bucket-name-2":["ak", "sk", "endpoint"]\...
               │    └ <_json.Scanner object at 0x000001C184D760E0>
               └ <json.decoder.JSONDecoder object at 0x000001C184D89000>

json.decoder.JSONDecodeError: Invalid control character at: line 6 column 29 (char 156)

**Comment by myhloli** [comment]
https://github.com/opendatalab/MinerU/issues/311
